sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -805,12 +906,14 @@ static int sof_card_dai_links_create(struct device *dev,
 #endif
 	struct snd_soc_dai_link_component *ssp_components;
 	struct snd_soc_acpi_mach_params *mach_params;
-	int ssp_num, sdw_num, hdmi_num = 0, dmic_num;
+	int ssp_num, sdw_num = 0, hdmi_num = 0, dmic_num;
 	const struct snd_soc_acpi_link_adr *adr_link;
 	struct snd_soc_dai_link_component *cpus;
+	bool group_generated[MAX_SDW_LINKS];
```

Comment:
Is it possible to have more group number than link number? ie. two group in one link?  Maybe MAX_GROUP is better than MAX_SDW_LINKS

Reply:
For real case, it is very rare for a case with at least 8 codecs more than 4 group, even 3 group.   Do we need to pay for it ?